server/src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -339,9 +372,15 @@ private ConfigProperties() {
             this.put(AMQP_KEYSTORE_PASSWORD, "password");
             this.put(AMQP_TRUSTSTORE, "/etc/candlepin/certs/amqp/candlepin.truststore");
             this.put(AMQP_TRUSTSTORE_PASSWORD, "password");
+            this.put(QPID_STARTUP_CHECK_ENABLED, "true");
+            this.put(SUSPEND_MODE_ENABLED, "true");
+            this.put(QPID_QMF_RECEIVE_TIMEOUT, "5000");
+            this.put(QPID_MODE_TANSITIONER_DELAY_GROWTH, "2");
+            this.put(QPID_MODE_TRANSITIONER_INITIAL_DELAY, "10");
+            this.put(QPID_MODE_TRANSITIONER_MAX_DELAY, "1200"); // 1200 seconds = 20 minutes
```

Comment:
20 minutes between checks is an awful long time. 5 minutes might be more appropriate.